audit/schema.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE decrypt_event (
+  id SERIAL PRIMARY KEY,
+  timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  username TEXT,
+  file TEXT
+);
+
+CREATE RULE decrypt_event_delete_protection AS ON DELETE TO decrypt_event DO INSTEAD NOTHING;
```

Comment:
Do we actually want this? The GRANT rules already prevent the sops user from doing anything else but inserting, and I can think of a legitimate reason to purge the DB from old records.